Modules/Quotas/test/Quotas.Application.Tests/Tests/Tiers/HandlerTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Backbone.Modules.Quotas.Application.IntegrationEvents.Incoming;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Tiers;
+public class HandlerTests
+{
+    private readonly ILogger<TierCreatedIntegrationEventHandler> _tiersLogger;
+
+    public HandlerTests() { }
+
+    [Fact]
+    public async void Returns_add_when_tiers_created()
```

Comment:
Rename this test to 'Successfully_creates_tier_after_consuming_integration_event'